sdk/core/azure-core/vcpkg.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "azure-core-cpp",
+    "version-string": "1.0.0",
```

Comment:
I see versions are `"1.0.0 + version-string instead of version-semver"` everywhere. Is this intentional? Just to have something so that vcpkg does not complain? Can we remove it?

Reply:
Can't remove it. it is required